.travis.yml

```diff
@@ -44,3 +48,19 @@ notifications:
 
 script:
   - make $MAKE_COMMAND -C package
+
+before_deploy:
+  - "mkdir -p upload"
+  - "mv package/*.tar.bz2 package/*.deb package/*.rpm package/*.zip upload || true"
+  - "rm -f upload/*source*"
```

Comment:
This is my hack to workaround the problem that we have several different builds in the build matrix, each of them produces different files and there was no good way to specify optional or possible files down in the deploy-step. This is just bash so I hope it works, I collect all the files I like to upload to `upload`.